README.md

```diff
@@ -3,6 +3,8 @@
 This tutorial was last given at SciPy 2018 in Austin Texas.
 [A video is available online](https://www.youtube.com/watch?v=mqdglv9GnM8).
 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/quasiben/dask-tutorial/add-binder)
```

Comment:
We should probably link this to the main repository before merging.  Also, should we point the link to JupyterLab with `?urlpath=lab` ?

Reply:
We may also want to change how this is styled a bit.  It think that most people today still don't know what binder is.  We might also want to increase the size of this, as we do in https://github.com/mrocklin/pydata-nyc-2018-tutorial#dask-tutorial-at-pydata-2018

Reply:
```suggestion
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dask/dask-tutorial/master?urlpath=lab)
```